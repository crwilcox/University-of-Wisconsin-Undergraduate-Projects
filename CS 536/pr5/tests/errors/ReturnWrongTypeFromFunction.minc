int IntFunction()
{
	return 2.0;
}

double DoubleFunction()
{
	return "String";
}

string StringFunction()
{
	return 666;
}

bool BoolFunction()
{
	return 123;
}
