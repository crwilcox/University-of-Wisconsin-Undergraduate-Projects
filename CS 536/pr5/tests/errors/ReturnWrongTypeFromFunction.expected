3:2:Bad return type
8:2:Bad return type
13:2:Bad return type
18:2:Bad return type
